packages/lib-react-components/src/Markdownz/Markdownz.stories.js

```diff
@@ -1,17 +1,11 @@
 import { Grid, Box, TableRow } from 'grommet'
-import styled from 'styled-components'
 
 import Markdownz from './Markdownz'
 import readme from './README.md'
 import markdownExample from '../../.storybook/lib/example.md'
 import { examples } from './helpers/storybookExamples'
 import markdownInGrid from './markdownGridExample.md'
 
-const TableRowWithBorder = styled(TableRow)`
-  border-top: solid thin black;
-  border-bottom: solid thin black;
-`
```

Comment:
This is the only case where `styled()` was used in FEM in a dev environment. There's no reason for libraries in FEM to install `styled-components` as a devDep so I refactored this Story with a different, but similar "with custom-styled components" test case.